osu.Game/Graphics/Containers/UserDimContainer.cs

```diff
@@ -24,9 +24,9 @@ public abstract class UserDimContainer : Container
         protected const double BACKGROUND_FADE_DURATION = 800;
 
         /// <summary>
-        /// Whether or not user-configured settings relating to brightness of elements should be ignored
+        /// Whether or not user-configured effect settings should be applied to this container.
         /// </summary>
-        public readonly Bindable<bool> IgnoreUserSettings = new Bindable<bool>();
+        public readonly Bindable<bool> ApplyUserSettings = new Bindable<bool>(true);
```

Comment:
Now this one reads pretty back to front, no? With a class name like `UserDimContainer` you'd expect this to read the inverse of what it is. I guess it's done this way to match the screen version, but doesn't feel right. Maybe just revert to the previous iteration.

Reply:
Yeah this is actually why I left it as a side commit, will revert back.